HW3/theatre.html

```diff
@@ -0,0 +1,330 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>Theatre</title>
+<link rel="stylesheet" href="reset.css">
+<style>
+    h1 {
+        margin: 0;
+        padding: 0;
+        font-size: 0;
+    }
+    .logo {
+        width: 140px;
+        height: 50px;
+    }
+    select {
+        border:0;
+    }
+    .header-a {
+        padding: 5px;
+        margin: 6px;
+        color: white;
+        text-align: center;
+    }
+    .header-a:link {
+        text-decoration: none;
+    }
+    .header-a:visited {
+        text-decoration: none;
+    }
+    .header-a:active {
+        text-decoration: none;
+    }
+    .header-a:hover {
+        text-decoration: none;
+        color: black;
+        background-color: whitesmoke;
+    }
+    .header {
+        display: flex;
+        justify-content: center;
+        background-color: #C92329;
+
+    }
+    .lang {
+        margin-left: 30px;
+        border-radius: 50px;
+        background-color: #b7b7b7;
+        -moz-appearance: none;
+        -webkit-appearance: none;
+   }
+    .header-text {
+        margin: 15px 0 0 0;
+    }
+    .picture {
+        width: 100%;
+    }
+    .carousel-wrapper {
+        position: relative;
+        overflow: hidden;
+    }
+    .carousel-item {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        padding: 25px 50px;
+        opacity: 0;
+    }
+    .arrow {
+           position: absolute;
+           top: 0;
+           display: block;
+           width: 50px;
+           height: 100%;
+           -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+           background: url("img/carousel-arrow-dark.png") 50% 50%/20px no-repeat;
+       }
+    .arrow-prev {
+         left:0;
+        -webkit-transform: rotate(180deg);
+        transform: rotate(180deg);
+     }
+    .arrow-next {
+        right: 0;
+    }
+    .light {
+        color: white;
+    }
+    .arrow {
+        background:url("img/carousel-arrow-light.png")50%50%/20px no-repeat;
+    }
+    [id^="target-item"] {
+        display:none;
+    }
+    .item-1 {
+        z-index:2;
+        opacity:1;
+    }
+    *:target~.item-1 {
+        opacity:0;
+    }
+    #target-item-1:target~.item-1 {
+        opacity:1;
+    }
+    #target-item-2:target~.item-2,#target-item-3:target~.item-3 {
+        z-index:3;
+        opacity:1;
+    }
+    nav {
+        font-size: 18px;
+        margin: 20px 100px 0 150px;
+        width: 17%;
+    }
+    .nav-a {
+        text-decoration:none;
+        color: black;
+    }
+    .nav-a:link {
```

Comment:
наступні 3 селектори можна об'єднати:
```suggestion
    .nav-a:link, .nav-a:visited, .nav-a:active, .nav-a:hover  {
```